app/views/browse/_common_details.html.erb

```diff
@@ -38,4 +38,4 @@
   <% end %>
 </div>
 
-<%= render :partial => "browse/tag_details", :object => common_details.tags %>
+<%= render :partial => "browse/tag_details", :object => common_details.tags, :locals => { :tag_details_object => common_details } %>
```

Comment:
How about renaming `tag_details_object` to `current_feature`? I had some difficulty reasoning about what it represented, due to the naming.